definitions/npm/express_v4.x.x/flow_v0.25.x-/express_v4.x.x.js

```diff
@@ -12,7 +12,7 @@ declare module 'express' {
   }
   declare class Request extends http$IncomingMessage mixins RequestResponseBase {
     baseUrl: string;
-    body: mixed;
+    body: any;
```

Comment:
`Object` guarantees, (as just 1 example), that you don't pass this value into something that's a `number`:

```js
function sum(a: number, b: number) { return a + b; }
sum(req.body); // oops!
```